### PR TITLE
Add topics document

### PR DESCRIPTION
**Pull request by resterle**

### CODE REVIEW

File: topics.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Collection of topics
+
+Here we collect all the topics for future meetups with potential speekers/lecturer.
```
Comment (curiosdevcookie) [inline topics.md:3]: sp_ea_kers/lecturer_s

File: topics.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+### TDD
+How to implement test driven design principles in elixir?
+
+Speeker:
+
+### DDD
+How to implement domain driven design principles in elixir?
+How does DDD relate to (Phoenix) contexts?
```
Comment (curiosdevcookie) [inline topics.md]: phoenix (consistently small-caps)

File: topics.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Speeker:
+
+### Elixir's type system and dialyzer
+How does the type system in the elixir world work?
+
+Speeker:
+
+### Scaling and updating
+How to run and release an elixir application at scale
```
Comment (curiosdevcookie) [inline topics.md]: Missing "?" at the end

File: topics.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Speeker:
+
+### How to structure elixir apps
+What options are there to structure elixir applications
```
Comment (curiosdevcookie) [inline topics.md]: Missing "?" at the end

File: topics.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+What is elixir at its core?
+
+Speeker: Raphael
+
+### Introduction to the Ash framework
+
+Speeker:
+
+### How to push elixir
+How to make elixir a more viable options for real projects?
```
Comment (curiosdevcookie) [inline topics.md]: "option" in singular

File: topics.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+What options are there to structure elixir applications
+
+Speeker:
+
+### Macros and the compiler
+What is elixir at its core?
+
+Speeker: Raphael
+
+### Introduction to the Ash framework
```
Comment (curiosdevcookie) [inline topics.md:42]: "ash" (consistently, either capitalized or not)

File: topics.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+Speeker:
+
+### How to push elixir
+How to make elixir a more viable options for real projects?
+
+Speeker:
+
+## Elixir beyond Phoenix web apps
+What arer usecases for elxir beyond the standard web apps?
```
Comment (curiosdevcookie) [inline topics.md]: "are" -> scratch the "r"

File: topics.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+### Mashine lerning (with Nx?)
```
Comment (curiosdevcookie) [inline topics.md]: "Machine learning"

File: topics.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+Speeker:
+
+
+### TDD
+How to implement test driven design principles in elixir?
+
+Speeker:
+
+### DDD
+How to implement domain driven design principles in elixir?
```
Comment (curiosdevcookie) [inline topics.md]: domain-driven design

File: topics.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+Here we collect all the topics for future meetups with potential speekers/lecturer.
+
+### Into OTP
+What is OTP and how does it relate to elixir and erlang?
+
+Speeker:
+
+
+### TDD
+How to implement test driven design principles in elixir?
```
Comment (curiosdevcookie) [inline topics.md]: test-driven development

File: topics.md (hunk 7 above)
Comment (curiosdevcookie) [inline topics.md]: "use cases"

Comment (curiosdevcookie) [inline topics.md]: "elixir" ("i" is missing)

**Review by curiosdevcookie — COMMENTED** [review]
- [ ]  "speeker" -> "speaker"
- [ ] consistency in the capitalization of names, e.g. erlang vs. Phoenix

File: topics.md (hunk 7 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 10 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 9 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 8 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 7 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 6 above)
Comment (resterle) [reply topics.md:42]: From what i saw its called "Ash" not "ash"

File: topics.md (hunk 5 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 4 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 3 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 2 above)
Comment (resterle) [reply topics.md]: done

File: topics.md (hunk 1 above)
Comment (resterle) [reply topics.md:3]: done

**Review by curiosdevcookie — APPROVED** [review]

**Pull request merged**